pebblo/app/pebblo-ui/src/util.js

```diff
@@ -15,10 +13,36 @@ const MONTHS = [
   "Dec",
 ];
 
-export const get_Formatted_Date = (date) => {
-  const newDate = new Date(date);
-  return `${newDate.getDate()} ${MONTHS[newDate.getMonth()]
-    } ${newDate.getFullYear()}`;
+export const get_Formatted_Date = (
```

Comment:
nit-pick: Function names are using a mix of camelCase and snake_case. Stick to one naming convention.

Reply:
Resolved